test/instructions_test.rb

```diff
@@ -0,0 +1,56 @@
+require 'test_helper'
+
+class DummyClass
+  # This is the class referenced in `./data/valid_instructions.yml`.
+  # scraper_test expects the instruction file to contain a class to test
+  # against, a url and a :to_h field containing the expected data.
+  # To test ScraperTest::Instructions, DummyClass does not need to implement
+  # #to_h.
+end
```

Comment:
This is still quite confusing, I'm afraid. The important thing here is that the only thing you're testing is that the file is _valid_, not that the values in it are "correct" for the supplied URL. 

I fear slightly however, that you're boxing yourself into a bit of a corner here. Presumably in the next PR you're about to add a version of `DummyClass` that _does_ implement a `to_h`, so that you can test the testing of it? If so it might be simpler to simply add a `to_h` method that simply maps to `id: 34`, so that the example case is more in keeping with actual usage, rather than needing a lengthly explanation as to why it's not, when it's about to be changed to be so anyway.